layouts/section/job-posting.en.html

```diff
@@ -1,145 +1,154 @@
-              id="email"
-              aria-required="true"
-              aria-describedby="email-required"
-            />
-            <span
-              id="email-required"
-              class="error-message validate-required-message"
-              style="display: none;"
-              >This field is required</span
-            >
-          </div>
-
-          <!-- Resume -->
-          <div class="form-group">
-            <label for="resume">Resume</label>
-            <input class="resume"
-              type="file"
-              id="resume"
-              name="resume"
-              accept="application/msword, application/pdf, text/plain" required
-            />
-          </div>
-
-          <!-- Body -->
-          <div class="form-group">
-            <label for="body"
-              >Tell us in 300 words or less why you want to work with us and,
-              <strong>if applicable</strong>, what we should know about your
-              work samples
-            </label>
-            <textarea
-              class="form-control validate-required validate-maxwords"
-              id="body"
-              name="body"
-              rows="5"
-              aria-required="true"
-              aria-describedby="body-required word-count-message"
-            ></textarea>
-            <span
-              id="body-required"
-              class="error-message validate-required-message"
-              style="display: none;"
-              >This field is required<br />
-            </span>
-            <span id="word-count-message" class="help-text">
-              <span id="word-count">300</span> words remaining</span
-            >
-          </div>
-          <!-- Samples of your work -->
-          <div class="form-group">
-            <label for="work_link">(Optional) Samples of your work (URLs, please!)</label>
-            <input
-              type="text"
-              class="form-control"
-              id="work_link"
-              name="work_link"
-            />
-          </div>
-          <div class="form-group">
-            <label class="wb-inv" for="work_link_2"
-              >Additional Work Samples</label
-            >
-            <input
-              type="text"
-              class="form-control"
-              id="work_link_2"
-              name="work_link_2"
-            />
-          </div>
-          <!-- Submit-->
-          <button type="submit" class="button" id="submitButton">
-            <span class="spinner" id="buttonSpinner"
-              ><i
-                style="display:inline-block;margin-right:10px;"
-                class="fas fa-spinner fa-spin"
-              ></i></span
-            >Submit
-          </button>
-          <a href="/legal/privacy-notice">Privacy Notice</a>
-        </form>
+        <h2 class="h2-font-weight">Our hiring process</h2>
+        <h1 class="cds-title heading-title"> At CDS, we welcome and invite difference.</h1>
```

Comment:
I'm curious as to why we have two `h1` tags on this page. 
While it's totally fine to have them from an a11y point of view even if it's discouraged on most GoC sites it can be confusing from a semantic point of view. 


Looking at the rendered page I'm thinking maybe we are missing something as having an `h2` with no content and then an h1 directly after is an a11y issue. Are we missing content or is the second h1 marked up improperly? 


![image](https://user-images.githubusercontent.com/87738/127054569-436d1182-a153-4d88-a367-2d45206035ed.png)

Reply:
Can we have multiple <h2> tags?